Lib/gftools/scripts/set_overlap_bits.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+from defcon import Font
+from pathops import Path
+from ufo2ft.filters.decomposeComponents import DecomposeComponentsFilter
+from ufo2ft.preProcessor import TTFPreProcessor
+import math
+from fontTools.designspaceLib import DesignSpaceDocument
+
+
+def set_overlap_bits(ufo):
+    # Skip setting bits if ufo
+    if any(g.lib.get("public.truetype.overlap") for g in ufo):
+        return
+    # Decompose components first because some component glyphs may have
+    # components that overlap each other
+    outline_glyphset = TTFPreProcessor(
+        ufo, filters=[DecomposeComponentsFilter()]
+    ).process()
+
+    overlaps = set()
+    for glyph in outline_glyphset.values():
+        skia_path = Path()
+        pen = skia_path.getPen()
+        for contour in glyph:
+            contour.draw(pen)
+        area = skia_path.area
+        # rm overlaps
+        skia_path.simplify()
```

Comment:
this can fail sometimes (pathops has some tricky unfixable bugs), I think it'd be better to default to setting the flag for all glyphs _except_ those whose area does _not_ change after simplifying